src/scss/index.scss

```diff
@@ -110,6 +112,13 @@ $color-selection-bg           : inherit;
 // <hr> border color
 $hr-color                     : $grey--lightest;
 
+// Alert messages
+$color-alert-message          : $blue;
+
+// Text descriptions
+$color-text-description       : $blue;
```

Comment:
I'd move this to be next to the other `$color-text--` definitions on lines 74-78 and rename it to be `$color-text--description`